src/common/dao/pgsql.go

```diff
@@ -96,6 +100,10 @@ func (p *pgsql) Register(alias ...string) error {
 		return err
 	}
 
+	db, _ := orm.GetDB(an)
+	db.SetConnMaxLifetime(p.connMaxLifetime)
```

Comment:
@sayaoailun Please review the code(line 98), there are some difference after beego upgrade.

Reply:
@chlins Fixed.